tsl/test/expected/bgw_db_scheduler.out

```diff
@@ -431,12 +432,54 @@ SELECT ts_bgw_params_reset_time();
 (1 row)
 
 DELETE FROM _timescaledb_config.bgw_job;
-SELECT insert_job('test_job_2', 'bgw_test_job_2_error', INTERVAL '100ms', INTERVAL '100s', INTERVAL '100ms');
+SELECT insert_job('test_job_1b', 'bgw_test_job_1', INTERVAL '100ms', INTERVAL '100s', INTERVAL '1s');
  insert_job 
 ------------
        1001
 (1 row)
 
+SELECT ts_bgw_db_scheduler_test_run_and_wait_for_scheduler_finish(25);
+ ts_bgw_db_scheduler_test_run_and_wait_for_scheduler_finish 
+------------------------------------------------------------
+ 
+(1 row)
+
+SELECT job_id, next_start, last_finish as until_next, last_run_success, total_runs, total_successes, total_failures, total_crashes
+FROM _timescaledb_internal.bgw_job_stat;
+ job_id |           next_start           |          until_next          | last_run_success | total_runs | total_successes | total_failures | total_crashes 
+--------+--------------------------------+------------------------------+------------------+------------+-----------------+----------------+---------------
+   1001 | Fri Dec 31 16:00:00.1 1999 PST | Fri Dec 31 16:00:00 1999 PST | t                |          1 |               1 |              0 |             0
+(1 row)
+
+SELECT * FROM sorted_bgw_log;
+ msg_no | application_name |                        msg                         
+--------+------------------+----------------------------------------------------
+      0 | DB Scheduler     | [TESTING] Registered new background worker
+      1 | DB Scheduler     | [TESTING] Wait until (RANDOM), started at (RANDOM)
+      0 | test_job_1b      | Execute job 1
+(3 rows)
```

Comment:
This do not show the debug output, but when looking in the log after running the test it shows up.

Reply:
Removed check to make debug output show up in the log (if the `log_min_message` variable is set appropriately).